analyses/gene-set-enrichment-analysis/run-gsea.sh

```diff
@@ -0,0 +1,25 @@
+####################################################
+# Stephanie J. Spielman for ALSF CCDL 2020
+#
+# Run the GSEA pipeline, currently just `01-conduct-gsea-analysis.R`
+# 
+# Usage: bash run-gsea.sh
+#
+# Takes a single environmental variable `OPENPBTA_SMALLSET`. If FALSE or 0 (default), runs all samples. If TRUE or 1 runs a subset of samples (for testing in CI)
+####################################################
+
+
+set -e
+set -o pipefail
+
+SMALLSET=${OPENPBTA_SMALLSET:-1}
```

Comment:
I would expect this to be 
```suggestion
SMALLSET=${OPENPBTA_SMALLSET:-0}
```
because we want it to be FALSE by default.

Reply:
I went back and forth on this for a while, ha. 
Although, if indeed only 21 samples are in CI at all, this is entirely unneeded. Would it be reasonable to just scrap the argument altogether?